src/fmu/dataio/dataio.py

```diff
@@ -396,7 +395,7 @@ class ExportData:
 
     # Need to store these temporarily in variables until we stop
     # updating state of the class also on export and generate_metadata
-    _classification: enums.Classification = enums.Classification.internal
+    _classification: Classification = Classification.internal
```

Comment:
I've been a bit random about when to import the module and when to import the class within the module. A basic heuristic is if it's just one or two things, I'll import those classes directly rather than the module. I don't claim this is the best way, it may in fact be worst...

Reply:
I think we all are a bit random on that :+1: However in this case I find the former a bit more intuitive, so we see directly that `Classification` here is "about" an `enum`?